rts/rts.c

```diff
@@ -1907,21 +1938,24 @@ int main(int argc, char **argv) {
     unsigned int seed;
     if (ddb_host) {
         GET_RANDSEED(&seed, 0);
+        log_info("Starting distributed RTS node, host=%s, node_id=%d, rack_id=%d, datacenter_id=%d\n", rts_host, rts_node_id, rts_rack_id, rts_dc_id);
         log_info("Using distributed database backend replication factor of %d\n", ddb_replication);
-        db = get_remote_db(ddb_replication);
-        for (int i=0; i<ddb_no_host; i++) {
-            char *host = strdup(ddb_host[i]);
+		char ** seed_hosts = (char **) malloc(ddb_no_host * sizeof(char *));
+		int * seed_ports = (int *) malloc(ddb_no_host * sizeof(int));
 
-            int port = ddb_port;
-            char *colon = strchr(host, ':');
+        for (int i=0; i<ddb_no_host; i++) {
+        		seed_hosts[i] = strdup(ddb_host[i]);
+        		seed_ports[i] = ddb_port;
```

Comment:
more tab indent... I'll stop commenting for tab indent :stuck_out_tongue_winking_eye:

Reply:
Fixed. Note that there are some tabs in that file that pre-date the commits in this PR, most also coming from DB client code. I'd rather fix that in a separate "indentation fix" PR to keep the set of lines in the RTS functionally affected by this PR clear. I've fixed all tabs that were introduced by changes in this PR.